titiler/endpoints/factory.py

```diff
@@ -1101,6 +1111,7 @@ def tile(
                         src_path.url,
                         reader=self.dataset_reader,
                         reader_options=self.reader_options,
```

Comment:
Do you want to remove passing the `reader_options` to the backend?

Reply:
no we still need this because https://github.com/developmentseed/cogeo-mosaic/blob/master/cogeo_mosaic/backends/base.py#L156